src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,45 @@
+package tasks;
+public class Event extends Todo {
```

Comment:
Java coding standard: Logical units should be separated by a blank line. 

You could leave a line between package definition and class definition, to adhere to the standard and improve readability.